docs/guides/sft.md

```
@@ -201,10 +203,59 @@ policy:
+      exclude_modules: []           # List of module names not to apply LoRa.
+      dim: 32                       # LoRA rank (r): controls adaptation capacity
+      alpha: 32                     # LoRA scaling factor (effective lr = alpha/dim)
+      dropout: 0.0                  # Dropout probability for LoRA layers
+      dropout_position: "pre"       # Dropout position: "pre" or "post"
+      lora_A_init_method: "xavier"  # Initialization method for lora A: "xavier" or "uniform"
+      lora_B_init_method: "zero"    # Initialization method for lora B: "zero"
+      a2a_experimental: false       # Enables the experimental All-to-All (A2A) communication strategy.
+      lora_dtype: None              # Weight's dtype
+```
+
+### Parameter Details
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)